src/lib.rs

```diff
@@ -76,30 +77,31 @@ impl Instruction {
     pub fn execute(&self, interp: &mut Interpreter) {
         match *self {
             Instruction::IncPtr => {
-                interp.ptr = interp.ptr + 1;
+                interp.ptr = interp.ptr + Wrapping(1);
             },
             Instruction::DecPtr => {
-                interp.ptr = interp.ptr - 1;
+                interp.ptr = interp.ptr - Wrapping(1);
             },
             Instruction::IncVal => {
-                interp.tape[interp.ptr] = interp.tape[interp.ptr] + 1;
+                interp.tape[interp.ptr.0] = interp.tape[interp.ptr.0] + Wrapping(1);
```

Comment:
`Wrapping` is a tuple struct so `.0` is the accessor (projection) for the first field of the struct, in this case the wrapped value.